compass/ocean/tests/soma/default/__init__.py

```diff
@@ -0,0 +1,81 @@
+from compass.testcase import TestCase
+from compass.ocean.tests.soma.initial_state import InitialState
+from compass.ocean.tests.soma.forward import Forward
+from compass.ocean.tests.soma.default.analysis import Analysis
+from compass.validate import compare_variables, compare_timers
+
+
+class Default(TestCase):
+    """
+    The default test case for the SOMA test group simply creates the mesh and
+    initial condition, then performs a short forward run with analysis members.
+
+    Attributes
+    ----------
+    resolution : str
+        The resolution of the test case
+    """
+
+    def __init__(self, test_group, resolution):
+        """
+        Create the test case
+
+        Parameters
+        ----------
+        test_group : compass.ocean.tests.soma.Soma
+            The test group that this test case belongs to
+
+        resolution : str
+            The resolution of the test case
+        """
+        name = 'default'
+        self.resolution = resolution
+        subdir = '{}/{}'.format(resolution, name)
+        super().__init__(test_group=test_group, name=name,
+                         subdir=subdir)
+
+        step = InitialState(test_case=self, resolution=resolution)
+        # add the namelist options specific ot this test case
+        package = 'compass.ocean.tests.soma.default'
+        for out_name in ['namelist_mark_land.ocean', 'namelist.ocean']:
+            step.add_namelist_file(package, 'namelist.init', mode='init',
+                                   out_name=out_name)
+        self.add_step(step)
+
+        with_particles = self.resolution == '32km'
```

Comment:
This should be the default but we want a config option for turning particles on or off at all resolutions.

Reply:
It turns out to be difficult to make this a config option. Primarily, this is because the analysis step shouldn't happen if you're not using particles.  So I've made 2 versions of the test case at 32km resolution instead, one with and one without particles.  I think that's okay for now and we could add particles at other resolutions if we want to.